### ISSUE TITLE
Admin\Create record confirmation message

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#1983]
#### Dependencies

- Blocked by
- Blocking

#### Description:

When a user adds an contact, they see a confirmation message (snackbar):
When a user adds a facility, they see a confirmation message (snackbar):
When a user adds an operator, they see a confirmation message (snackbar):

#### Acceptance Criteria:

Given I am on the add contact page
When I have filled out the form fields and clicked submit
Then I am shown a SnackBar

Given I am on the add facility page
When I have filled out the form fields and clicked submit
Then I am shown a SnackBar
~Then I am brought back to the Facilities page~

Given I am on the add operator page
When I have filled out the form fields and clicked submit
Then I am shown a SnackBar

#### Development Checklist:

- [ ] In `bciers/libs/components/src/form/SingleStepTaskListForm.tsx` modify the `onSubmit` to `  setIsSnackbarOpen(true);` for all submits
- [ ]  In `bciers/libs/components/src/form/SingleStepTaskListForm.tsx` modify Snackbar option ` message=` to "All changes have been successfully saved""
- [ ] Add create confirmation message test inbciers/libs/components/src/form/SingleStepTaskListForm.test.tsx`
- [ ] Add create confirmation message test in `ContactForm.test.tsx`
- [ ] Add create confirmation message test in `FacilityForm.test.tsx`
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by patriciarussellCAS** [comment]
Tagging @zoeyli-46 so we can discuss this ticket. Should a wireframe be created for this? I can help refine this ticket once we have that, if that's helpful.

**Comment by zoeyli-46** [comment]
@patriciarussellCAS done, I linked to the wireframe and removed the wireframe needed label!

**Comment by shon-button** [comment]
@patriciarussellCAS 

As `bc-cas-dev`,  verify snackbar displays after creating the following documents:
![Image](https://github.com/user-attachments/assets/99c4ef11-cf41-4e4a-b427-f1f88096b68f)

https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/contacts/add-contact
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/operations/002d5a9e-32a6-4191-938c-2c02bfec592d/facilities/add-facility?operations_title=Operation%202
